roles/uperf-bench/tasks/main.yml

```diff
@@ -81,30 +81,33 @@
   - name: Start Client(s)
     k8s:
       definition:
-        kind: Pod
-        apiVersion: v1
+        kind: Job
+        apiVersion: batch/v1
         metadata:
           name: '{{ meta.name }}-uperf-client-{{item.status.podIP}}'
           namespace: '{{ meta.namespace }}'
-          labels:
-            app : uperf-bench-client
         spec:
```

Comment:
Should we use : `ttlSecondsAfterFinished` to automatically cleanup?

Reply:
Yes I think that'll be helpful, also I was thinking we can clean up server as soon as the client job completes successfully. Thus we'll keep the client pod but delete the serve pod once client finishes running. what'd you think?

Reply:
Hm - I would think the other way around. Client completes, deletes. Server deletes? Not sure if the order really matters...

Reply:
After discussion with @jtaleric on bluejeans, we've agreed upon deleting the server once the client job completes and use the `ttlSecondsAfterFinished` for client job.